integrationtests/tests/tests.rs

```diff
@@ -235,8 +235,8 @@ async fn wallet_peg_ins_that_are_unconfirmed_are_rejected() -> Result<()> {
 
 #[tokio::test(flavor = "multi_thread")]
 async fn wallet_peg_outs_must_wait_for_available_utxos() -> Result<()> {
-    test(2, |fed, user, bitcoin, _, _| async move {
-        // at least one epoch needed to establish fees
+    non_lightning_test(2, |fed, user, bitcoin, _, _| async move {
+        // at least one epoch needed to estabilish fees
```

Comment:
nit: this is introducing a typo, it was correct before :)

Reply:
:) will fix!